packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -39,6 +39,9 @@ export const GRID_ROW_LEAVE = 'rowLeave';
 export const GRID_ROW_EDIT_MODEL_CHANGE = 'editRowModelChange';
 export const GRID_ROW_SELECTED = 'rowSelected';
 
+export const GRID_COLUMN_HEADER_FOCUS = 'columnHeaderFocus';
+export const GRID_COLUMN_HEADER_NAVIGATION_KEYDOWN = 'columnHeaderNavigationKeyDown';
```

Comment:
GRID_COLUMN_HEADER_NAVIGATION_KEY_DOWN?

Reply:
Based on this line of thinking, other constants should be changed as well:

```diff
- export const GRID_CELL_DOUBLE_CLICK = 'cellDoubleClick';
+ export const GRID_CELL_DBLCLICK = 'cellDblclick';
```

We have to agree on a single naming strategy and stick to it. Currently, the constants file has different patterns. My preference is for `GRID_COLUMN_HEADER_NAVIGATION_KEYDOWN` as I think that each spoken word should be preceded by an underscore (except for the first word). However, `GRID_COLUMN_HEADER_NAVIGATION_KEYDOWN` is also fine if we make it clear that the DOM event must be used as suffix _as it is_.

Obviously, updating all constants is out of scope of this PR. I think this is something https://github.com/mui-org/material-ui-x/issues/1301 should address too.

Reply:
+1 for: 

- consistency
- outside of the scope of this PR
- opening an RFC with a proposal
- present tense (this is an event)
- staying close to the event name (e.g. `document.addEventListener('dbclick')` for the dom, but prop in React is `onDoubleClick` so 🤷‍♂️)

Reply:
In that case I would push the change `GRID_COLUMN_HEADER_NAVIGATION_KEY_DOWN` into a separate one as well.

Reply:
I think that there are two levels to this discussion. The first one is about having the string match the format of the const. Here, it's a new line, and should match.

Then, there is a larger discussion about the whole naming consistent of the costs, which can happen with an RFC.

Reply:
Agree, I went with this one `export const GRID_COLUMN_HEADER_NAVIGATION_KEYDOWN = 'columnHeaderNavigationKeydown';`